telos-backend/src/routes/api/text.js

```diff
@@ -0,0 +1,34 @@
+const { json } = require('express');
```

Comment:
Is this import necessary? It's the same `require()` as the line below

Reply:
You're right, this was accidental!